back-end/courses/serializers.py

```
@@ -23,12 +42,42 @@ class Meta:
+    
+    class Meta:
+        model = Course
+        fields = [
+            "tutor",
+            "category",
+            "title",
+            "thumbnail_image_url",
+            "description",
+            "curriculum",
+            "max_tutees"
+        ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)